src/components/App.js

```diff
@@ -1,68 +0,0 @@
-import React, { useEffect, useState } from "react";
-import LocationDetails from "./LocationDetails";
-import ForecastSummaries from "./ForecastSummaries";
-import ForecastDetails from "./ForecastDetails";
-import SearchForm from "./SearchForm";
-import getForecast from "../requests/getForecast";
-import "../styles/App.css";
-
-function App() {
-  const [selectedDate, setSelectedDate] = useState(0);
-  const [forecasts, setForecasts] = useState([]);
-  const [location, setLocation] = useState({
-    city: "",
-    country: "",
-  });
-  const [searchText, setSearchText] = useState("");
-  const [errorMessage, setErrorMessage] = useState("");
-
-  const selectedForecast = forecasts.find(
-    (forecast) => forecast.date === selectedDate
-  );
-
-  useEffect(() => {
-    getForecast(setSelectedDate, setForecasts, setLocation);
-  }, []);
-
-  const handleCitySearch = () => {
-    getForecast(
-      setSelectedDate,
-      setForecasts,
-      setLocation,
-      searchText,
-      setErrorMessage
-    );
-  };
-
-  const handleForecastSelect = (date) => {
-    setSelectedDate(date);
-  };
-
-  return (
-    <div className="App">
-      <LocationDetails
-        city={location.city}
-        country={location.country}
-        errorMessage={errorMessage}
-      />
-      <SearchForm
-        searchText={searchText}
-        setSearchText={setSearchText}
-        onSubmit={handleCitySearch}
-      />
-      <br />
```

Comment:
I would add margin or padding to the bottom of the search form hear rather than using a line break. Try and only use line breaks when you need to move something to the next line. They aren't a very precise way of spacing containers